src/layouts/shortcodes/cloud-prepare/aws.md

```diff
@@ -6,20 +6,18 @@ Please ensure that the AWS Region you deploy to supports this instance type.
 Alternatively, you can choose to deploy using a different instance type.</p>
 </div>
 
-You need to extract your infra ID and region in order to use the command.
-Its possible to extract them from the `metadata.json` file that the OpenShift Installer created:
-
-```bash
-metadata_file=<path/to>/metadata.json
-infra_id=$(jq -r .infraID $metadata_file)
-region=$(jq -r .aws.region $metadata_file)
-```
+You need to extract your infra ID and region for each cluster.
+It's possible to extract them from the `metadata.json` files that the OpenShift Installer created.
```

Comment:
It's also possible to pass the metadata.json file as an argument and subctl will extract the necessary info. But this description is good aswell.

Reply:
That seems simpler, I will update the docs to that method.

Reply:
Done, and yes I think that makes things much simpler. Thanks @sridhargaddam!